src/WorkOS.net/Services/SSO/Enums/ProviderType.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace WorkOS
+{
+    using System.Runtime.Serialization;
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Converters;
+
+    /// <summary>
+    /// Constants that enumerate available OAuth provider types to initiate OAuth authentication.
+    /// </summary>
+    [JsonConverter(typeof(StringEnumConverter))]
+    public enum ProviderType
```

Comment:
Authkit isn't here because this is just for the standalone SSO API. When we add user management APIs, we'll make another ProviderType.